nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/dbcp/hive/HiveConnectionPool.java

```diff
@@ -426,4 +445,13 @@ public String getConnectionURL() {
     boolean isAllowExplicitKeytab() {
         return Boolean.parseBoolean(System.getenv(ALLOW_EXPLICIT_KEYTAB));
     }
+
+    private long extractMillisWithInfinite(PropertyValue prop) {
+        if (prop.getValue() == null || "-1".equals(prop.getValue())) {
```

Comment:
Minor adjustment to reuse the static default:
```suggestion
        if (prop.getValue() == null || DEFAULT_MAX_CONN_LIFETIME.equals(prop.getValue())) {
```